persistence-jpa/javadsl/src/main/resources/reference.conf

```diff
@@ -18,4 +18,10 @@ lagom.persistence.jpa {
 }
 //#persistence
 
+
+db.default {
+  # Configure the default database to be bound to JNDI
+  jndiName = DefaultDS
+}
+
```

Comment:
We only bind the JNDI data source for JPA. This must still be tested on a real app and we should also assess the impact of that. 

All code using Slick directly will use the `AsyncExecuter` but JPA code will bypass it and use the `DataSource` directly. Not yet clear what will the impact of this. We need to better understand what happens inside Slick. If JPA consumes a connection from the pool, will it impact the `AsyncExecutor`?

Reply:
Yeah, I'm not sure. JPA is running on the Slick execution context, so depending on how Slick is using the `maxConnections` parameter this may or may not be a problem. In any case, I don't think it's a _new_ problem so I think as long as this PR is strictly better than what exists now we can merge and then resolve other issues separately.